FND/Agreements/Contracts.rdf

```diff
@@ -325,43 +333,40 @@ A unilateral contract differs from a Bilateral Contract, in which the parties ex
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
-		<skos:definition>the date on which a contract has been fully performed by all parties</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom rdf:resource="https://definitions.uslegal.com/e/executed-contract/"/>
-		<fibo-fnd-utl-av:explanatoryNote>In other words, the execution date is the date on which the terms of the contract have been completely fulfilled.</fibo-fnd-utl-av:explanatoryNote>
+		<skos:definition>indicates the date on which a contract has been signed by all the necessary parties</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>This may or may not be the &apos;effective date&apos; of the contract, which may be specified in the body of the document.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasExecutionDateTimeStamp">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDateTimeStamp"/>
 		<rdfs:label>has execution date time stamp</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;DateTimeStamp"/>
-		<skos:definition>the date and time, including time zone, on which a contract has been fully performed by all parties</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom rdf:resource="https://definitions.uslegal.com/e/executed-contract/"/>
-		<fibo-fnd-utl-av:explanatoryNote>In other words, the execution date time stamp is the date and time on which the terms of the contract have been completely fulfilled.</fibo-fnd-utl-av:explanatoryNote>
+		<skos:definition>indicates the date and time, including time zone, on which a contract has been signed by all the necessary parties</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasGoverningJurisdiction">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;isGovernedBy"/>
 		<rdfs:label>has governing jurisdiction</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-agr-ctr;Contract"/>
 		<rdfs:range rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
-		<skos:definition>the jurisdiction governing the contract, as agreed by all parties</skos:definition>
+		<skos:definition>indicates the jurisdiction governing the contract, as agreed by all parties</skos:definition>
 		<skos:editorialNote>As modeled, this relationship combines two slightly different senses in which a Jurisdiction may be named in some Contract: the jurisdiction under whose laws the contract is deemed to be in force, and the jurisdiction under which the parties agree to submit in the event of any dispute resolution. ScopeNote: One thing to tease out is whether &apos;Dispute Resolution&apos; and other forms of &apos;Governing Law&apos; are one and the same thing or not. Dispute Resolution is uncontroversial, the question is whether there are other implications to Governing Law or if it&apos;s the same thing. For instance I may undertake to behave as though I were responsible to a particular authority i.e., a particular set of statutes.</skos:editorialNote>
```

Comment:
If we don't have one already we should have a new issue to resolve this ambiguity

Reply:
Please raise another issue for this, if you think that there is ambiguity there.

Reply:
It's the editorial note in the ontology itself which pointed out the ambiguity. 
I assume by your request you're saying there is no existing issue for this.

Reply:
I think we need a new issue - it's not clear whether we need a second property.  My understanding is that the jurisdiction is the one whose laws govern how any dispute will be handled and when required, where the contract is legally recorded

Reply:
I haven't changed this ... please think about whether we should remove the note or do something else - add another property

Reply:
Raised #833